src/exportAuthRecord.ts

```diff
@@ -0,0 +1,117 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IActionContext } from '@microsoft/vscode-azext-utils';
+import * as fs from 'fs-extra';
+import * as os from 'os';
+import * as path from 'path';
+import * as vscode from 'vscode';
+import { ext } from './extensionVariables';
+
+
+/**
+ * Registers the exportAuthRecord callback for session changes and ensures the auth record is exported at least once on activation.
+ */
+import { callWithTelemetryAndErrorHandling } from '@microsoft/vscode-azext-utils';
+import type { ExtensionContext } from 'vscode';
```

Comment:
These two imports should be merged into the rest of them above.

Reply:
Updated.
import type { ExtensionContext } from 'vscode', remains as separate import.